apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridView.swift

```diff
@@ -45,8 +46,24 @@ struct SpeakerGridView: View {
                 GeometryReader { geometry in
                     LazyVGrid(columns: columns, spacing: spacing) {
                         ForEach($viewModel.onscreenSpeakers, id: \.self) { $speaker in
-                            SpeakerVideoView(speaker: $speaker)
-                                .frame(height: geometry.size.height / CGFloat(rowCount) - spacing)
+                            Group {
+                                if role == .host && !speaker.isMuted && !speaker.isYou  {
+                                    Menu(
+                                        content: {
+                                            Button(
+                                                action: { viewModel.muteSpeaker(speaker) },
+                                                label: { Label("Mute", systemImage: "mic.slash") }
```

Comment:
Have you guys thought of the mute button disabling feature by admin?

Reply:
@piyushtank what specific functionality are you referring to?

The functionality in this PR allows a host to mute any other speaker very similar to the live audio app.

Reply:
+1 Per our slack discussion